src/man/sssd-ldap.5.xml

```diff
@@ -1654,6 +1655,22 @@ ldap_access_filter = (employeeType=admin)
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>ldap_ppolicy_pwd_change_threshold (integer)</term>
+                    <listitem>
+                        <para>
+                            Forces a password change when server side password policy
+                            controls are enabled and remaining grace logins go below
+                            the threshold. Note that the minimum useful value is 3, as
+                            changing the password consumes 3 grace logins (initial login,
+                            current password verification and the password change).
+                        </para>
+                        <para>
+                            Default: true
```

Comment:
Hi,

thanks for adding the option. The default currently is `0` not `true`.

Since the say "remaining grace logins go below the threshold" and the comparison in `check_pwexpire_ldap()` is `ppolicy->grace < pwd_change_thold` I think " the minimum useful value is 4" for the new `ldap_ppolicy_pwd_change_threshold` option. Maybe it would be better to say "reaches the threshold or goes below" and use `<=`  in the comparison to avoid confusion?

bye,
Sumit

Reply:
Hi Sumit, I have updated the documentation, I hope it is clearer now.

If `ppolicy->grace <= pwd_change_thold` the minimum useful value is 2 because where there are 3 remaining logins you consume 1 to authenticate, then the server returns 2 remaining in the bind response, the condition is true, password change is triggered and those 2 are consumed to check the current password and perform the change.

Reply:
Hi,

thank you for the update.

bye,
Sumit